docusaurus/docs/reference/tunnelers/60-linux/50-linux-tunnel-troubleshooting.mdx

```diff
@@ -15,12 +15,11 @@ The tunneler obeys the value of `logLevel` in `/var/lib/ziti/config.json`. The i
 
 Create a log file from the current systemd service invocation to share with collaborators.
 
-```bash
+```text
```

Comment:
for something titled "increase log level" it feels out of place to tell people to create a log. also it'd be really great if you could have compassion for the reviewer and stop formatting commands like this. i sat here and tried to diff the command to make sure it wasn't changing, it was a waste of 30s.

Reply:
How would you prefer to format things like this? BTW I looked at this example and found an issue and also decided to simplify it in the follow-on pull request: https://github.com/openziti/ziti-doc/pull/867